docs/ops/movement/GatherTree_1.md

```diff
@@ -8,21 +8,34 @@
 
 **Detailed description**
 
-GatherTree operation implements the same algorithm as GatherTree operation in TensorFlow. Please see complete documentation [here](https://www.tensorflow.org/versions/r1.12/api_docs/python/tf/contrib/seq2seq/gather_tree?hl=en).
+GatherTree operation implements the same algorithm as GatherTree operation in TensorFlow.
+Please see complete documentation [here](https://www.tensorflow.org/addons/api_docs/python/tfa/seq2seq/gather_tree).
```

Comment:
--> `The GatherTree operation implements the same algorithm as the [GatherTree operation in TensorFlow](https://www.tensorflow.org/addons/api_docs/python/tfa/seq2seq/gather_tree).`

Reply:
fixed